index.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+const Logr = require('logr');
+const _ = require('lodash');
+const defaultOptions = {
+  // time in ms, longer than this will trigger a warning:
+  threshold: 1000,
+  // you can pass optiosn to customize how logr notifies you of a slow response:
+  logr: {
+  }
+};
+
+// request timeouts are indexed by request.id:
+const outstandingTimeouts = {};
+
+exports.register = (server, config, next) => {
+  const options = _.defaults(config, defaultOptions);
+  const log = new Logr(options.logr);
+
+  // when a request's timeout expires, do this:
+  const requestTimeoutExpired = (request) => {
+    // log the tardiness:
+    log(`Request ${request.id} lagging more than ${options.threshold} seconds`);
```

Comment:
lets make this a object with a message `{ id, message: '...', threshold: 500, url: /blah, userAgent: '' }`